### PR TITLE
OIE Updates - Nodejs SDK Self-Registration Use Case

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Updates to the Node.js self-registration [use-case](https://developer.okta.com/docs/guides/oie-embedded-sdk-use-cases/nodejs/oie-embedded-sdk-use-case-self-reg/#summary-of-steps) including improved sequence diagrams, nodejs-sample-app-specific images, alt-text updates, and minor updates to the integration steps.
- **Is this PR related to a Monolith release?** n/a

### Resolves:

* [OKTA-408181](https://oktainc.atlassian.net/browse/OKTA-408181)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/nodejs/integrationsteps.md [hunk 1]
```diff
@@ -1,21 +1,26 @@
 ### Step 1: Create a sign-up link for new users
 
-The self-registration flow begins when the user clicks a **Sign up** link. On the sign-in page, create a **Sign up** link that links to the create account page that you create in the next step,  as shown in the following example.
-
+The self-registration flow begins when the user clicks the **Sign Up** link (or **Register** from the SDK sample application home page). On the sign-in page, create a **Sign Up** link that links to the create account page that you create in the next step,  as shown in the following example.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/nodejs/integrationsteps.md:3]:
```suggestion
The self-registration flow begins when the user clicks the **Sign Up** link (or **Register** from the SDK sample application home page). On the sign-in page, create a **Sign Up** link that links to the create account page that you create in the next step, as shown in the following example.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/nodejs/integrationsteps.md [hunk 2]
```diff
@@ -1,21 +1,26 @@
...
 
 </div>
 
-> **Note:** The **Sign up** link appears under the **Continue** button.
+> **Note:** The **Sign Up** link appears under the **Forgot your password?** link.
 
 ### Step 2: Enter information in the create account page
 
-The next step is to enter basic information (for example, email, first, and last name). Create a page that accepts this information. The following shows an example of a create account page.
+The next step is to enter basic information (for example, email, first, and last name). Create a page that accepts this information. The following image shows an example of a create account page.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/nodejs/integrationsteps.md:15]:
```suggestion
The next step is to enter basic information (for example, email, first name, and last name). Create a page that accepts this information. The following example shows a create account page.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/nodejs/integrationsteps.md [hunk 3]
```diff
@@ -51,31 +56,63 @@ If the org's application is properly configured with multiple factors, `idx.regi
 
 ### Step 3: User enters password authenticator and password
 
-After the user enters the password authenticator value, and `idx.register` is called with this value, the response returns a status of `Idx.Status:PENDING` and a `nextStep` field requiring a password value. The user is directed to a page to enroll a password value, as shown in the SDK sample application route to `/enroll-or-reset-password-authenticator`.
+Create a page that displays an authenticator enrollment selection. In this example, password is configured.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/nodejs/integrationsteps.md:59]:
```suggestion
Create a page that displays an authenticator enrollment selection. In this example, the password is configured.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/nodejs/integrationsteps.md [hunk 4]
```diff
@@ -51,31 +56,63 @@ If the org's application is properly configured with multiple factors, `idx.regi
...
+
+<div class="common-image-format">
+
+![Displays a Select authenticator page screenshot that includes a selection for password. A Select button continues the flow.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-self-serv-screen-enroll-pwd-auth-nodejs.png)
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/nodejs/integrationsteps.md:63]:
```suggestion
![Displays a Select authenticator page screenshot that includes a selection for the password. A Select button continues the flow.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-self-serv-screen-enroll-pwd-auth-nodejs.png)
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/nodejs/integrationsteps.md [hunk 5]
```diff
@@ -1,21 +1,26 @@
...
+The next step is to enter basic information (for example, email, first, and last name). Create a page that accepts this information. The following image shows an example of a create account page.
+
+<div class="common-image-format">
+
+![Displays a Self-Service Registration page screenshot that includes fields for the first name, last name and email address. A Register button starts the self-registration flow. ](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-self-serv-screen-enroll-user-nodejs.png)
+
+</div>
 
-When the user clicks **Register**, pass the user profile data captured from the Create account page into the `idx.register` method, as shown in the `register.js` page of the SDK sample application:
+When the user clicks **Register**, pass the user profile data captured from the Create account page into the `idx.register` method, as shown in `register.js` of the SDK sample application:
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/nodejs/integrationsteps.md:23]:
```suggestion
When the user clicks **Register**, pass the user profile data that is captured from the Create account page into the `idx.register` method, as shown in `register.js` of the SDK sample application:
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/nodejs/integrationsteps.md [hunk 6]
```diff
@@ -51,31 +56,63 @@ If the org's application is properly configured with multiple factors, `idx.regi
...
-After the user enters the password authenticator value, and `idx.register` is called with this value, the response returns a status of `Idx.Status:PENDING` and a `nextStep` field requiring a password value. The user is directed to a page to enroll a password value, as shown in the SDK sample application route to `/enroll-or-reset-password-authenticator`.
+Create a page that displays an authenticator enrollment selection. In this example, password is configured.
+
+<div class="common-image-format">
+
+![Displays a Select authenticator page screenshot that includes a selection for password. A Select button continues the flow.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-self-serv-screen-enroll-pwd-auth-nodejs.png)
+
+</div>
+
+After the user enters the password authenticator value, and `idx.register` is called with this value, the response returns a status of `Idx.Status:PENDING` and a `nextStep` field requiring the user to enter a password value. The user is directed to a page to set up a password value, as shown in the SDK sample application route to `/enroll-authenticator`.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/nodejs/integrationsteps.md:67]:
```suggestion
After the user enters the password authenticator value, and `idx.register` is called with this value, the response returns a status of `Idx.Status:PENDING` and a `nextStep` field that requires the user to enter a password value. The user is directed to a page to set up a password value, as shown in the SDK sample application route to `/enroll-authenticator`.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/nodejs/integrationsteps.md [hunk 7]
```diff
@@ -51,31 +56,63 @@ If the org's application is properly configured with multiple factors, `idx.regi
...
+![Displays a Select authenticator page screenshot that includes a selection for email. A Select button continues the flow.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-self-serv-screen-enroll-email-auth-nodejs.png)
+
+</div>
+
 ### Step 5: User selects email authenticator
 
 In this use case, the user selects the **Email** factor as the authenticator to verify. Pass this user-selected authenticator to `idx.register`.
 
- If the call is successful, the method returns a status of `Idx.Status:PENDING` and a `nextStep` field requiring verification, which indicates that the SDK is ready for the email verification code. The next step is to redirect the user to the email verification code page, as shown in the SDK sample application route to `/authenticator`.
+ If the call is successful, the method returns a status of `Idx.Status:PENDING` and a `nextStep` field requiring verification, which indicates that the SDK is ready for the email verification code. The next step is to redirect the user to the email verification code page, as shown in the SDK sample application route to `/enroll-authenticator`.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/nodejs/integrationsteps.md:91]:
```suggestion
 If the call is successful, the method returns a status of `Idx.Status:PENDING` and a `nextStep` field that requires verification, which indicates that the SDK is ready for the email verification code. The next step is to redirect the user to the email verification code page, as shown in the SDK sample application route to `/enroll-authenticator`.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/nodejs/integrationsteps.md [hunk 8]
```diff
@@ -51,31 +56,63 @@ If the org's application is properly configured with multiple factors, `idx.regi
...
+
+<div class="common-image-format">
+
+![Displays an Enroll email authenticator page screenshot that includes field for the email verification code. A Verify button continues the flow.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-self-serv-screen-email-verify-nodejs.png)
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/nodejs/integrationsteps.md:95]:
```suggestion
![Displays an Enroll email authenticator page screenshot that includes the field to enter the email verification code. A Verify button continues the flow.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-self-serv-screen-email-verify-nodejs.png)
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/nodejs/integrationsteps.md [hunk 9]
```diff
@@ -51,31 +56,63 @@ If the org's application is properly configured with multiple factors, `idx.regi
...
+
+</div>
+
 If the user decides to skip the optional factor, they are considered signed in since they have already verified the required factors. See [Step 8, Option 1: Skip phone factor](#step-8-option-1-skip-phone-factor) for the skip authenticator flow. If the user decides to select the optional factor, see [Step 8, Option 2: User selects phone authenticator](#step-8-option-2-user-selects-phone-authenticator) for the optional phone authenticator flow.
 
 ### Step 7, Option 1: Skip phone factor
 
-If the user decides to skip the phone factor enrollment, make a call to `idx.register` passing in the value `{skip: true}`. This method skips the authenticator enrollment, as shown on the `Authenticator.js` from the SDK sample application.
+If the user decides to skip the phone factor enrollment, make a call to `idx.register` passing in the value `{skip: true}`. This method skips the authenticator enrollment, as shown in `Authenticator.js` from the SDK sample application.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/nodejs/integrationsteps.md:115]:
```suggestion
If the user decides to skip the phone factor enrollment, make a call to `idx.register` that passes the value `{skip: true}`. This method skips the authenticator enrollment, as shown in `Authenticator.js` in the SDK sample application.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/nodejs/integrationsteps.md [hunk 10]
```diff
@@ -90,17 +127,31 @@ If the request to skip the optional authenticator is successful, the SDK returns
...
+![Displays an Enroll phone authenticator page screenshot that includes a selection for Phone verification method (SMS) and a field for the phone number. A Next button continues the flow.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-self-serv-screen-enroll-phone-auth-nodejs.png)
 
-The Auth JS SDK sends the phone authenticator data to Okta. Otka processes the request and sends an SMS code to the specified phone number. After the SMS code is sent, Okta sends a response to the SDK, which returns `Idx.Status:PENDING` and the `nextStep` field requiring verification. This status indicates that the user needs to provide the verification code for the phone authenticator, as shown in the SDK sample application route to `/authenticator`.
+</div>
+
+### Step 8, Option 2: User selects SMS verify method and enters phone number
+
+The user can selects SMS or voice verification to receive the phone verification code. Capture this information and send it to`idx.register`. In this use case, use SMS. The user then enters the phone number where the SMS code is sent.
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/nodejs/integrationsteps.md:140]:
```suggestion
The user can select SMS or voice verification to receive the phone verification code. Capture this information and send it to`idx.register`. In this use case, use SMS. The user then enters the phone number where the SMS code is sent.
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/nodejs/integrationsteps.md [hunk 11]
```diff
@@ -90,17 +127,31 @@ If the request to skip the optional authenticator is successful, the SDK returns
...
+
+>**Note:** The SDK requires that the phone number follows the `+########` format, which starts with a (+) sign. See [Data Requirements - Phone number](/docs/guides/oie-embedded-sdk-common/nodejs/main/#phone-number).
+
+The SDK sends the phone authenticator data to Okta, processes the request, and sends an SMS code to the specified phone number. After the SMS code is sent, Okta sends a response to the SDK, which returns `Idx.Status:PENDING` and the `nextStep` field requiring verification. This status indicates that the user needs to provide the verification code for the phone authenticator:
+
+<div class="common-image-format">
+
+![Displays an Enroll phone authenticator page screenshot that includes a field to enter phone (SMS) verification code. A Verify button continues the flow.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-self-serv-screen-enroll-phone-verify-nodejs.png)
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/nodejs/integrationsteps.md:148]:
```suggestion
![Displays an Enroll phone authenticator page screenshot that includes a field to enter the phone (SMS) verification code. A Verify button continues the flow.](/img/oie-embedded-sdk/oie-embedded-sdk-use-case-simple-self-serv-screen-enroll-phone-verify-nodejs.png)
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/nodejs/summaryofsteps.md [hunk 12]
```diff
@@ -1,6 +1,6 @@
 # Summary of steps
 
-The sequence of steps for self-service registration is described in the following three sequence diagrams:
+The sequence of steps for self-service registration is described in the following sequence diagrams:
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/nodejs/summaryofsteps.md:3]:
```suggestion
The following sequence diagrams describe the sequence of steps for self-service registration:
```

File: packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/nodejs/integrationsteps.md (hunk 9 above)
Comment (brianduffield-okta) [reply packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-self-reg/nodejs/integrationsteps.md:115]: I'm going to not commit this suggestion, although the passage could be better worded. I.e calling the method doesn't pass the value; you need to call the method with the value (passed in).

**Pull request merged**